### PR TITLE
Host - UI: Scenario 3 e2e test

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/25103
- bcgov/entity/issues/25096

*Description of changes:*
- scenario 3 e2e test
- fix docs being deleted unintentionally
- several empty pdfs for testing doc upload

https://github.com/user-attachments/assets/1d7c96f1-5d1e-4140-aa38-eabfb20651db

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the BC Registry and Digital Services BSD 3-Clause License

### CODE REVIEW

File: strr-host-pm-web/app/composables/useHostPmModals.ts
Comment (deetz99) [inline strr-host-pm-web/app/composables/useHostPmModals.ts:1]: moved docs reset from watcher into here, watcher was causing unintended side effect

File: strr-host-pm-web/app/pages/default.vue
Comment (deetz99) [inline strr-host-pm-web/app/pages/default.vue:1]: dont need index page

File: strr-host-pm-web/tests/e2e/host-smoke.test.ts
Comment (deetz99) [inline strr-host-pm-web/tests/e2e/host-smoke.test.ts:1]: e2e test file

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-424-zicptkhd.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**